test/test_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,43 @@
+import os.path
+
+import PIL.Image
+import numpy
+import pytest
+import torch
+
+import torchvision.ops
+
+ASSETS_DIRECTORY = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
+
+
+@pytest.fixture
+def labeled_image() -> torch.Tensor:
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "labeled_image.png")) as image:
+        return torch.tensor(numpy.array(image, numpy.int))
+
+
+@pytest.fixture
+def masks() -> torch.Tensor:
```

Comment:
unless we're planning on using t his fixture in other tests, I would suggest to not make it a fixture. It could just be within the `test_masks_to_bounding_boxes`